### PR TITLE
feat(order-status): Order Status Page.

### PR DESCRIPTION
**Pull request by kibo-geetanshuchhabra**

### CODE REVIEW

File: components/KiboAppHeader.vue [hunk 1]
```diff
@@ -59,7 +62,7 @@
           </div>
           <div class="kibo-top-bar__content"></div>
           <div class="kibo-top-bar__nav-link">
-            <div><SfMenuItem label="Nav Link 1" /></div>
+            <div><SfMenuItem label="Order Status" @click="goToOrderStatus" /></div>
```
Comment (kibo-chandradeeptalaha) [inline components/KiboAppHeader.vue]: localization ?

File: components/KiboOrderItemDetails.vue [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
-            :name="$t('Order Total:')"
-            :value="$n(orderTotal, 'currency')"
-            class="sf-order-summary__property"
-          />
+          <SfProperty :name="$t('Order Total')" class="sf-order-summary__property">
+            <template #value>
+              <span class="sf-property__value">
+                {{ $n(orderTotal, "currency") }}
+                <span class="items">{{
+                  `(${orderItemCount} item${orderItemCount > 1 ? $t("s") : $t("")})`
```
Comment (kibo-chandradeeptalaha) [inline components/KiboOrderItemDetails.vue]: follow this https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument

File: components/KiboStoreAddress.vue [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  <div>
+    <div v-for="location in storeDetails" :key="location.code">
+      <KiboStoreDetails :location="location" :is-order-detail="true" />
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { useAsync } from "@nuxtjs/composition-api"
+import { useStoreLocations } from "@/composables"
+import { storeLocationGetters } from "~~/lib/getters"
```
Comment (kibo-chandradeeptalaha) [inline components/KiboStoreAddress.vue]: use @

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
please check

File: components/KiboStoreDetails.vue [hunk 4]
```diff
@@ -105,21 +118,60 @@ export default defineComponent({
 .sf-accordion-item {
   &__header {
     text-decoration: underline;
+
+    @include for-mobile {
+      padding: var(--spacer-sm) var(--spacer-sm) var(--spacer-sm) 0;
+    }
```
Comment (kibo-sushant) [inline components/KiboStoreDetails.vue:124]: Default style should be mobile only

File: lib/getters/orderGetters.ts [hunk 5]
```diff
@@ -26,6 +26,19 @@ const getOrderTotal = (order: Order) => order?.total
 
 const getOrderStatus = (order: Order) => order?.status
 
+const getShippedTo = (order: Order) =>
+  order?.fulfillmentInfo?.fulfillmentContact?.firstName.charAt(0).toUpperCase() +
+  order?.fulfillmentInfo?.fulfillmentContact?.firstName.slice(1) +
+  " " +
+  order?.fulfillmentInfo?.fulfillmentContact?.lastNameOrSurname.charAt(0).toUpperCase() +
+  order?.fulfillmentInfo?.fulfillmentContact?.lastNameOrSurname.slice(1)
```
Comment (kibo-sushant) [inline lib/getters/orderGetters.ts]: avoid code repetation
order?.fulfillmentInfo?.fulfillmentContact?.firstName
order?.fulfillmentInfo?.fulfillmentContact?.lastNameOrSurname

File: pages/order-status/index.vue [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+    <div class="order-status-title">
+      <div class="header-text-weight">{{ $t("View Order Status") }}</div>
+    </div>
+    <hr class="order-status-hr" />
+    <div v-show="!isOpenOrderStatus" class="order-status">
+      <div class="status-order">
+        {{ $t("Checking the status of your order is fast and simple!") }}
+      </div>
+      <div class="track-order">
+        {{ $t("Simply enter your order number and billing email to track your order.") }}
```
Comment (kibo-sushant) [inline pages/order-status/index.vue]: Have you updated lang/en.js file? I don't see this file in PR

File: components/KiboAppHeader.vue (hunk 1 above)
Comment (kibo-geetanshuchhabra) [reply components/KiboAppHeader.vue]: Resolved

File: components/KiboOrderItemDetails.vue (hunk 2 above)
Comment (kibo-geetanshuchhabra) [reply components/KiboOrderItemDetails.vue]: Resolved

File: components/KiboStoreAddress.vue (hunk 3 above)
Comment (kibo-geetanshuchhabra) [reply components/KiboStoreAddress.vue]: Resolved

File: components/KiboStoreDetails.vue (hunk 4 above)
Comment (kibo-geetanshuchhabra) [reply components/KiboStoreDetails.vue:124]: Resolved

File: lib/getters/orderGetters.ts (hunk 5 above)
Comment (kibo-geetanshuchhabra) [reply lib/getters/orderGetters.ts]: Resolved

File: pages/order-status/index.vue (hunk 6 above)
Comment (kibo-geetanshuchhabra) [reply pages/order-status/index.vue]: Resolved

**Review by kibo-sushant — APPROVED** [review]

File: lang/en.js [hunk 7]
```diff
@@ -234,4 +234,20 @@ export default {
   "is an invalid code": "is an invalid code",
   "promo code applied": "promo code applied",
   Qty: "Qty:",
+  "Checking the status of your order is fast and simple!":
+    "Checking the status of your order is fast and simple!",
+  "Simply enter your order number and billing email to track your order":
```
Comment (TriptiVermaXP) [inline lang/en.js]: Please keep _key_ on camel case as much as possible instead whole text.
eg. enterOrderNumberAndBillingEmail

**Review by TriptiVermaXP — APPROVED** [review]
Approving the PR with small comment, please check.

File: lang/en.js (hunk 7 above)
Comment (kibo-kevinwatts) [reply lang/en.js]: Keys should also be short, the key used here is the full length of the message

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
shorten the keys in lang/en file

File: lang/en.js (hunk 7 above)
Comment (kibo-geetanshuchhabra) [reply lang/en.js]: Resolved

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**